### PR TITLE
TRUNK-5672 | Add liquibase changesets to populate username for admin user

### PR DESCRIPTION
**Pull request by mddubey**
As part of this PR following things are done:
- [x] Add a new file `liquibase-update-to-2.4.xml`.
- [x] Included the `liquibase-update-to-2.4.xml` in `liquibase-update-to-latest.xml` file.
- [x] Add the changeset to `liquibase-update-to-2.4.xml`.
- [x] Add the changeset to `liquibase-update-to-2.3.xml`.

Jira ticket: https://issues.openmrs.org/browse/TRUNK-5672
Talk Thread: https://talk.openmrs.org/t/admin-user-doesnt-have-a-username/25145/

### CODE REVIEW

File: api/src/main/resources/liquibase-update-to-2.3.xml [hunk 1]
```diff
@@ -108,4 +108,16 @@
 			<column name="description" type="varchar(1)" />
 		</createTable>
 	</changeSet>
+	<changeSet id="TRUNK-5672-201910211105" author="Mritunjay">
```
Comment (dkayiwa) [inline api/src/main/resources/liquibase-update-to-2.3.xml]: We do not put the changeset in both the 2.3 and 2.4 files. Just one of the two.

File: api/src/main/resources/liquibase-update-to-2.3.xml [hunk 2]
```diff
@@ -108,4 +108,16 @@
...
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				SELECT COUNT(*) FROM users where username = 'admin'
```
Comment (dkayiwa) [inline api/src/main/resources/liquibase-update-to-2.3.xml]: Wouldn't it better to search by uuid?

File: api/src/main/resources/liquibase-update-to-2.3.xml (hunk 1 above)
Comment (mddubey) [reply api/src/main/resources/liquibase-update-to-2.3.xml]: Sure, it makes sense! I just got confused. I will remove `2.4` and just add it to `2.3`  file

File: api/src/main/resources/liquibase-update-to-2.3.xml (hunk 2 above)
Comment (mddubey) [reply api/src/main/resources/liquibase-update-to-2.3.xml]: In the `pre-condition`, I wanted to say that if there is already a user with username `admin` I would not run this changeset and mark it as RAN.
If I will put the pre-condition using `uuid` the count will always come as true. In that case, I will never execute the `changeset`.

Comment (mddubey) [reply api/src/main/resources/liquibase-update-to-2.3.xml]: The other part is I am not really sure, where are we populating `uuid` to `admin` user. In [this changeset](https://github.com/openmrs/openmrs-core/blob/a7600d359821debe312d1d14b49cd4714481d7c0/api/src/main/resources/liquibase-core-data.xml#L52), we are not populating any `uuid` at all. 

My worry would be if it is generated as `uuid()` function call, we will get different `UUIDs` on different instances.

I see we generate a hardcoded UUID for `daemon` user in [this changeset](https://github.com/openmrs/openmrs-core/blob/a7600d359821debe312d1d14b49cd4714481d7c0/api/src/main/resources/liquibase-core-data.xml#L1078), but not for `admin` user.

File: api/src/main/resources/liquibase-update-to-2.3.xml (hunk 1 above)
Comment (mddubey) [reply api/src/main/resources/liquibase-update-to-2.3.xml]: Fixed this.

File: api/src/main/resources/liquibase-update-to-2.3.xml (hunk 2 above)
Comment (dkayiwa) [reply api/src/main/resources/liquibase-update-to-2.3.xml]: Have you tried to test this change by running the compiled web application with it?

Comment (mddubey) [reply api/src/main/resources/liquibase-update-to-2.3.xml]: A question on that, I have been trying to run it my local. This is what I tried
- `mvn clean install` 
- Copy the generated `openmrs.war` to docker
- Restart the container

For some reason, it is not running the changeset. I couldn't see it in the `liquibasechangelog` table.

Am I missing some step?

Comment (dkayiwa) [reply api/src/main/resources/liquibase-update-to-2.3.xml]: Can you change to the `webapp` directory and then run this command? `mvn jetty:run`

Comment (mddubey) [reply api/src/main/resources/liquibase-update-to-2.3.xml]: Thanks, I tried this and I see username for `admin` user is populated.
The query:
```
select system_id, username from users;
```
Gives result:
```
admin, admin
daemon, daemon
```

And the query:
```
select ID,DATEEXECUTED,Comments from liquibasechangelog where author = 'Mritunjay';
```
Gives result:
```
'TRUNK-5672-201910211105','2019-10-23 08:24:55','Populate username for admin user'
```

**Pull request merged**